apps/server/src/modules/provisioning/uc/provisioning.uc.ts

```diff
@@ -1,41 +1,55 @@
-import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { Injectable, InternalServerErrorException, UnprocessableEntityException } from '@nestjs/common';
 import { SystemUc } from '@src/modules/system/uc/system.uc';
 import { SystemProvisioningStrategy } from '@shared/domain/interface/system-provisioning.strategy';
-import { ProvisioningStrategy } from '@src/modules/provisioning/strategy/base.strategy';
 import { Logger } from '@src/core/logger';
 import { ProvisioningSystemInputDto } from '@src/modules/provisioning/dto/provisioning-system-input.dto';
 import { ProvisioningSystemInputMapper } from '@src/modules/provisioning/mapper/provisioning-system-input.mapper';
-import { PlaceholderProvisioningStrategy } from '@src/modules/provisioning/strategy/placeholder/placeholder.strategy';
+import { SanisProvisioningStrategy, SanisStrategyData } from '@src/modules/provisioning/strategy/sanis/sanis.strategy';
+import { IservProvisioningStrategy, IservStrategyData } from '@src/modules/provisioning/strategy/iserv/iserv.strategy';
+import { ProvisioningDto } from '@src/modules/provisioning/dto/provisioning.dto';
 
 @Injectable()
 export class ProvisioningUc {
 	constructor(
 		private readonly systemUc: SystemUc,
```

Comment:
A UC should not use other UC

Reply:
At the time of creating this UC it was decided that UCs can use each other. Until we have a definitive decision in the ARC Group we should not refactor our UCs every PR to fit the current consensus.

Reply:
solved this. Now services just call other servies. no UCs involved anymore